phpBB/adm/style/acp_styles.html

```diff
@@ -32,6 +32,10 @@
 
 <!-- IF L_EXPLAIN --><p>{L_EXPLAIN}</p><!-- ENDIF -->
 
+<fieldset class="quick">
+	<span class="small"><a href="https://www.phpbb.com/customise/db/styles-2" target="_blank">{L_BROWSE_STYLES_DATABASE}</a></span>
```

Comment:
OK but we should only link to the "official" real phpBB Titania IMO. All other links back to "official" phpBB are always to phpbb.com and not another IST AFAIK.